src/topology/algebra/uniform_group.lean

```diff
@@ -154,8 +154,25 @@ uniform_continuous_of_tendsto_one $
   (hu : cauchy_seq u) (hv : cauchy_seq v) : cauchy_seq (u * v) :=
 uniform_continuous_mul.comp_cauchy_seq (hu.prod hv)
 
+@[to_additive] lemma cauchy_seq.mul_const {ι : Type*} [semilattice_sup ι] [nonempty ι]
+  {u : ι → α} {x : α} (hu : cauchy_seq u) : cauchy_seq (λ n, u n * x) :=
+uniform_continuous_mul.comp_cauchy_seq (hu.prod $ cauchy_seq_const x)
```

Comment:
This way you don't need `nonempty` assumption (actually, it follows from `cauchy_seq u`).
```suggestion
@[to_additive] lemma cauchy_seq.mul_const {ι : Type*} [semilattice_sup ι]
  {u : ι → α} {x : α} (hu : cauchy_seq u) : cauchy_seq (λ n, u n * x) :=
(uniform_continuous_id.mul uniform_continuous_const).comp_cauchy_seq hu
```